### PR TITLE
Enable SwiftLint on CI

### PR DESCRIPTION
**Pull request by giginet**
Enable linting on CI

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 language: objective-c
 osx_image: xcode10.2
+before_script:
+  - swiftlint --strict
```
Comment (giginet) [inline .travis.yml]: If linter causes violations then CI will fail.

File: Sources/Tools/DumpRuleDocument.swift [hunk 2]
```diff
@@ -34,7 +35,11 @@ struct DumpRuleDocument: CommandProtocol {
     func run(_ options: Options) -> Result<(), ClientError> {
         let content = Rules.allRules.map { $0.dumpMarkdown() }.joined(separator: "\n\n")
         if let path = options.path {
-            try! content.write(toFile: path, atomically: true, encoding: .utf8)
+            do {
+                try content.write(toFile: path, atomically: true, encoding: .utf8)
```
Comment (giginet) [inline Sources/Tools/DumpRuleDocument.swift]: Avoid `force_try` violation

**Comment by giginet** [comment]
@kateinoigakukun 

Screen Shot 2019-04-17 at 14 50 13

By the way, I can't merge after approval.
Please refer settings 🙏

**Comment by kateinoigakukun** [comment]
@giginet I changed branch setting. Could you try again?

**Comment by giginet** [comment]
Screen Shot 2019-04-17 at 14 58 46

There is no merge button 😭

**Comment by giginet** [comment]
Works! ✨

**Comment by kateinoigakukun** [comment]
I forgot to give you "write" permission 🙃

**Pull request merged**